lighthouse-core/gather/gatherers/css-usage.js

```diff
@@ -5,35 +5,40 @@
  */
 'use strict';
 
-const Gatherer = require('./gatherer.js');
+const Gatherer = require('../../fraggle-rock/gather/base-gatherer.js');
 
 /**
  * @fileoverview Tracks unused CSS rules.
  */
 class CSSUsage extends Gatherer {
```

Comment:
Conceptually, I don't think this should be a snapshot gatherer. Let's flag it in the FR issue for discussion and a potential rewrite soon?